client/src/components/FlavorThumbnail.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from "./flavor-thumbnail.module.css";
```

Comment:
would be good to merge the FlavorThumbnail PR first and pull staging to this branch... or only merge this branch and cancel the FlavorThumbnail PR since this has both PR essentially

Reply:
@larkinds should we close this PR since it was already merged as a part of another PR?

Reply:
This was not merged as another PR, it is separate. It just built on that other PR